tests/password_manager/test_add_primary_password.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+from selenium.webdriver import Firefox
+from selenium.webdriver.support import expected_conditions as EC
+
+from modules.page_object import AboutPrefs
+from modules.util import BrowserActions
+
+
+@pytest.fixture()
+def test_case():
+    return "2245178"
+
+
+def test_add_primary_password(driver: Firefox):
+    """
+    C2245178: Verify that a primary password can be added in about:preferences#privacy
+    """
+    # Instantiate objects
+    about_prefs = AboutPrefs(driver, category="privacy").open()
+    ba = BrowserActions(driver)
```

Comment:
Would love if this was just something like `browser_actions` but I suspect it has been that way for a while now.

Reply:
yeah, all the tests i've seen use ```ba``` so I think i'll keep it like that